### PR TITLE
Initialize `governance-policy-addon-controller`

### PR DESCRIPTION
**Pull request by dhaiducek**

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        - controller
+        image: policy-addon-image
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: CONFIG_POLICY_CONTROLLER_IMAGE
+          value: quay.io/open-cluster-management/config-policy-controller:latest
```
Comment (mprahl) [inline config/manager/manager.yaml:42]: These images need to be changed to the -io ones.

File: pkg/addon/policyframework/manifests/managedclusterchart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  operator: Exists
+  effect: NoSchedule
+
+clusterName: null
+
+global: 
+  imagePullPolicy: IfNotPresent
+  imagePullSecret: open-cluster-management-image-pull-credentials
+  imageOverrides: 
+    governance_policy_spec_sync: quay.io/open-cluster-management/governance-policy-spec-sync:latest
```
Comment (mprahl) [inline pkg/addon/policyframework/manifests/managedclusterchart/values.yaml:39]: These image references need to use the "-io" org.

**Comment by dhaiducek** [comment]
> I'd like us to set up the "correct" GitHub fork relationship for this sooner rather than later. Thoughts?

Just that I hadn't thought about how to do that yet. 😉 "Replacing it with a new fork" sounds risky, though any method probably involves risk and an abundance of caution. Unless you have a plan in mind, I'd have to look into it...

**Comment by dhaiducek** [comment]
Considering the forking, I think it makes sense to force-push commits from `stolostron` to the `main` branch and then use this PR to show/review the differences. Just a min...or an hour. Or tomorrow. Who knows. 🕐 😆

**Comment by dhaiducek** [comment]
Looks like GitHub was smart enough to close this to prevent catastrophe! Nice. `main` has been updated and updates continue in:
- #2

**Pull request closed without merging**